### DISCUSSION TITLE
Show tab buffer

### DISCUSSION THREAD

**Discussion by im6h** [helix-editor/helix#3326] [Ideas] [answered]
I think the tab buffer is a great feature. `Space -  b` shows all buffer I moved but the tab buffer is fast with `g-n` or `g-p`

![Screen Shot 2022-08-04 at 17 31 27](https://user-images.githubusercontent.com/35485655/182826287-876224f7-ceb8-4d37-b656-ff25260177c7.png)

**Comment by the-mikedavis** [comment, marked as answer]
There's a PR for a bufferline: https://github.com/helix-editor/helix/pull/2759

**Reply by im6h** [reply]
Thank you. That is greate
